### PR TITLE
Updated tags for MCR images

### PR DESCRIPTION
**Pull request by yuslepukhin**
**Description**:
Added 1.4.0 tags and updated latest for CPU and CUDA MCR images.

### CODE REVIEW

**Comment by yuslepukhin** [comment]
Looks like we are building lots of code for README.md update

File: dockerfiles/README.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ Use `docker pull` with any of the images and tags below to pull an image and try
 
 | Build Flavor      | Base Image                            | ONNX Runtime Docker Image tags        | Latest                  |
 |-------------------|---------------------------------------|---------------------------------------|-------------------------|
-| Source (CPU)      | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0, :v0.5.0, v0.5.1, :v1.0.0, :v1.2.0, :v1.3.0     | :latest                 |
-| CUDA (GPU)        | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0-cuda10.0-cudnn7, :v0.5.0-cuda10.1-cudnn7, :v0.5.1-cuda10.1-cudnn7, :v1.0.0-cuda10.1-cudnn7, :v1.2.0-cuda10.1-cudnn7, :v1.3.0-cuda10.1-cudnn7 | :latest-cuda            |
+| Source (CPU)      | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0, :v0.5.0, v0.5.1, :v1.0.0, :v1.2.0, :v1.3.0, :v1.4.0     | :latest                 |
+| CUDA (GPU)        | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0-cuda10.0-cudnn7, :v0.5.0-cuda10.1-cudnn7, :v0.5.1-cuda10.1-cudnn7, :v1.0.0-cuda10.1-cudnn7, :v1.2.0-cuda10.1-cudnn7, :v1.3.0-cuda10.1-cudnn7, :v1.4.0-cuda10.1-cudann7 | :latest-cuda            |
```
Comment (faxu) [inline dockerfiles/README.md:26]: Is it a typo for cudnn?

File: dockerfiles/README.md [hunk 2]
```diff
@@ -22,8 +22,8 @@ Use `docker pull` with any of the images and tags below to pull an image and try
...
 | TensorRT (x86)    | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0-tensorrt19.03, :v0.5.0-tensorrt19.06, :v1.0.0-tensorrt19.09, :v1.2.0-tensorrt20.01, :v1.3.0-tensorrt20.01 | :latest-tensorrt        |
```
Comment (jywu-msft) [inline dockerfiles/README.md:27]: can you add v1.4.0-tensorrt20.01 tag as well? 
I just published it.

Comment (jywu-msft) [reply dockerfiles/README.md:27]: published openvino images as well. so the following tags can be added as well
```
v1.4.0-openvino-2020.3.194-cpu
v1.4.0-openvino-2020.3.194-gpu
v1.4.0-openvino-2020.3.194-myriad
v1.4.0-openvino-2020.3.194-vadm
```

File: dockerfiles/README.md (hunk 1 above)
Comment (yuslepukhin) [reply dockerfiles/README.md:26]: Yes.

---
In reply to: [458435711](https://github.com/microsoft/onnxruntime/pull/4574#discussion_r458435711) [](ancestors = 458435711)

File: dockerfiles/README.md (hunk 2 above)
Comment (yuslepukhin) [reply dockerfiles/README.md]: Have you also advanced latest tags for them?

---
In reply to: [458782750](https://github.com/microsoft/onnxruntime/pull/4574#discussion_r458782750) [](ancestors = 458782750)

Comment (yuslepukhin) [reply dockerfiles/README.md]: @jywu-msft I have removed VAD-M installation instruction that pointed to 2020.2 version

File: dockerfiles/README.md [hunk 3]
```diff
@@ -175,19 +175,17 @@ Therefore, ONNX RT Execution Provider for **nGraph** will be deprecated starting
 
 ### OpenVINO on VAD-M Accelerator Version
 
-1.  Download OpenVINO **Full package** for version **2020.2** for Linux on host machine from [this link](https://software.intel.com/en-us/openvino-toolkit/choose-download) and install it with the help of instructions from [this link](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux.html)
+1. Install the drivers on the host machine according to the reference in [here](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux_ivad_vpu.html)
 
-2. Install the drivers on the host machine according to the reference in [here](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux_ivad_vpu.html)
-
-3. Build the docker image from the DockerFile in this repository.
+2. Build the docker image from the DockerFile in this repository.
```
Comment (jywu-msft) [inline dockerfiles/README.md:180]: why was step 2 removed? it is needed for running vad-m

File: dockerfiles/README.md [hunk 4]
```diff
@@ -175,19 +175,17 @@ Therefore, ONNX RT Execution Provider for **nGraph** will be deprecated starting
 
 ### OpenVINO on VAD-M Accelerator Version
 
-1.  Download OpenVINO **Full package** for version **2020.2** for Linux on host machine from [this link](https://software.intel.com/en-us/openvino-toolkit/choose-download) and install it with the help of instructions from [this link](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux.html)
+1. Install the drivers on the host machine according to the reference in [here](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux_ivad_vpu.html)
```
Comment (jywu-msft) [inline dockerfiles/README.md:178]: can you please revert the previous instructions and instead change to 2020.3 from 2020.2

File: dockerfiles/README.md [hunk 5]
```diff
@@ -175,19 +175,17 @@ Therefore, ONNX RT Execution Provider for **nGraph** will be deprecated starting
...
 
-2. Install the drivers on the host machine according to the reference in [here](https://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux_ivad_vpu.html)
-
-3. Build the docker image from the DockerFile in this repository.
+2. Build the docker image from the DockerFile in this repository.
      ```
       docker build --rm -t onnxruntime-vadm --build-arg DEVICE=VAD-M_FP16 --network host .
      ```
-4. Run hddldaemon on the host in a separate terminal session using the following command: 
+2. Run hddldaemon on the host in a separate terminal session using the following command:
```
Comment (jywu-msft) [inline dockerfiles/README.md:184]: the numbering looks off now. there's two step 2's.
i would just revert everything and for step 1, only change 2020.2 to 2020.3

**Pull request merged**